### DISCUSSION TITLE
Strange error when using   fltk.rs   native file dialog.

### DISCUSSION THREAD

**Discussion by jtreagan** [fltk-rs/fltk-rs#1581] [Q&A]
I added FLTK's file dialog to my project and it has really helped.  However, when I use the `NativeFileChooser` along with the `BrowseSaveFile` setting I get some weird errors that show up in my terminal.  I'll copy the errors below, but here's is the calling function:
```
 pub fn file_browse_save_fltr(extension: &str) -> String{
        let mut dialog = dialog::NativeFileChooser
            ::new(dialog::NativeFileChooserType
            ::BrowseSaveFile);
        dialog.set_filter(extension);
        dialog.show();
        let path = dialog.filename().to_str().unwrap().to_string();
        path
    }
```
And here are some of the errors:
```
(qbnk_rbld3:45550): Gtk-CRITICAL **: 14:35:13.865: gtk_tree_view_scroll_to_cell: assertion 'tree_view->priv->tree != NULL' failed

(qbnk_rbld3:45550): Gtk-CRITICAL **: 14:35:13.882: gtk_tree_view_scroll_to_cell: assertion 'tree_view->priv->tree != NULL' failed

(qbnk_rbld3:45550): Gtk-CRITICAL **: 14:35:13.898: gtk_tree_view_scroll_to_cell: assertion 'tree_view->priv->tree != NULL' failed

(qbnk_rbld3:45550): Gtk-CRITICAL **: 14:35:13.915: gtk_tree_view_scroll_to_cell: assertion 'tree_view->priv->tree != NULL' failed
Quitting 
```
Could anyone tell me what this is all about and how to fix it?  Thanks.

Note:
    I'm using the POP OS 22.04 version of Linux.  Since the error shows the involvement of `GTK`, I'm sure Gnome is somehow involved in this.  Also, the code executed just fine.  My data saved using the file name & path I entered.  I just don't know where these errors are coming from.

**Comment by MoAlyousef** [comment]
I'm not sure what the reason might be. I tried similar code on my system (ubuntu 22.04) and didn't get any warnings.
I used to get other warnings when showing a gtk dialog however it turned out a mismatch in gtk libs. It might be the same issue here.
